site/_includes/example.html

```diff
@@ -18,6 +18,21 @@
 
 {%- if include.hide_markup == null -%}
   {%- highlight html -%}
-    {{- include.content | replace: 'data-src="holder.js', 'src="...' -}}
+    {%- if include.content contains 'src="data:image/svg+xml' -%}
+      {%- assign modified_content = include.content
+        | replace: 'src="data:image/svg', 'src="✂️data:image/svg'
```

Comment:
I'm gonna backport this to a new PR to fix the holder.js replacement in v4-dev, because I'm not sure when this PR will be ready.